build.gradle

```diff
@@ -265,14 +265,15 @@ tasks.jpackage {
 	appVersion = new Date().format("yy.MM")
```

Comment:
Don't we want to use our rev number here?

Reply:
Can we instead use yy.MM.dd? Or 'yy.MM.' + version or something that gives us finer granularity?